src/GeositeFramework/js/SinglePluginModeHelp.js

```diff
@@ -19,11 +23,15 @@
         // place for it to be.
         $('#show-single-plugin-mode-help').click(function() {
             $container.show();
```

Comment:
If you move the `$activePlugin = $('.sidebar:visible');` assignment to the click handler, but above the `$container.show`, it finds it and hides it appropriately.  After `$container` is visible, that selector would find two items (help & plugin).